utils/helpers.py

```diff
@@ -0,0 +1,11 @@
+import random
+
+
+def get_random_number(min_value: int, max_value: int) -> int:
+    """
+    Returns random integer from a specified range
+    :param min_value: minimal range limit
+    :param max_value: maximal range limit
+    :return:
+    """
+    return random.randint(min_value, max_value)
```

Comment:
Overthinking, your function does not make sense in such an implementation, you just add another layer for out of box functionality. 

You can import random on a module layer so it will be available in every python file